packages/backend/src/openapi/webhooks.yaml

```
@@ -0,0 +1,230 @@
+  outgoingPaymentFailed:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment failed and won't be retried. Funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPaymentEvent'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  webMonetization:
+    post:
+      requestBody:
+        description: Notify account servicing entity that a Web Monetization payment was received and funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/webMonetizationEvent'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+
+components:
+  schemas:
+    incomingPaymentEvent:
+      required:
+        - id
+        - type
+        - data
+      properties:
+        id:
+          type: string
+          format: uuid
+        type:
+          type: string
+          enum:
+            - incoming_payment.completed
+            - incoming_payment.expired
+        data:
+          type: object
+          required:
+            - id
+            - paymentPointerId
+            - createdAt
+            - updatedAt
+            - expiresAt
+            - receivedAmount
+            - completed
+          properties:
+            id:
+              type: string
+              format: uuid
+            paymentPointerId:
+              type: string
+              format: uuid
+            completed:
+              type: boolean
+            incomingAmount:
+              $ref: ./schemas.yaml#/components/schemas/amount
+            receivedAmount:
+              $ref: ./schemas.yaml#/components/schemas/amount
+            description:
+              type: string
+            externalRef:
+              type: string
+            createdAt:
+              type: string
+              format: date-time
+            updatedAt:
+              type: string
+              format: date-time
+            expiresAt:
+              type: string
+              format: date-time
+    outgoingPaymentEvent:
+      required:
+        - id
+        - type
+        - data
+      properties:
+        id:
+          type: string
+          format: uuid
+        type:
+          type: string
+          enum:
+            - outgoing_payment.created
+            - outgoing_payment.completed
+            - outgoing_payment.failed
+        data:
+          type: object
+          required:
+            - id
+            - paymentPointerId
+            - createdAt
+            - updatedAt
+            - expiresAt
+            - sendAmount
+            - receiveAmount
+            - state
+            - stateAttempts
+            - balance
```

```suggestion
            - balance
            - receiver
```
https://github.com/interledger/rafiki/blob/7ca16474d5d901d25287572effc97fe06bb11498/packages/backend/src/open_payments/payment/outgoing/model.ts#L213

---

Good catch!